lib/logstash/outputs/elasticsearch/common_configs.rb

```diff
@@ -91,24 +91,12 @@ def self.included(mod)
       # Any special characters present in the URLs here MUST be URL escaped! This means `#` should be put in as `%23` for instance.
       mod.config :hosts, :validate => :uri, :default => [::LogStash::Util::SafeURI.new("//127.0.0.1")], :list => true
 
-      # This plugin uses the bulk index API for improved indexing performance.
-      # This setting defines the maximum sized bulk request Logstash will make.
-      # You may want to increase this to be in line with your pipeline's batch size.
-      # If you specify a number larger than the batch size of your pipeline it will have no effect,
-      # save for the case where a filter increases the size of an inflight batch by outputting
-      # events.
+      # DEPRECATED. This option reduces the maximum bulk size sent to ES in one shot, but is no longer necessary
```

Comment:
The docs generator looks at the `:deprecated` setting and should make a special note automatically without needing hand-written text here.